### ISSUE TITLE
Current master can't list older clusters

### ISSUE THREAD

**Issue by jaley** [Azure/aztk#466]
Since pulling from master (5761a3663adb261f49e0e7fc2301b327ee833c12), I can only see newly created clusters in `aztk spark cluster list`, but none of our long-running instances deployed with v0.6.0. If I checkout v0.6.0, I can see them again, so I can't pinpoint the exact commit, but at some point between 0.6.0 and the commit above, something happened to stop Aztk listing older clusters.

**Comment by emlyn** [comment]
Looks like it's because of: 1c31335fb05fb2c6586b40d66a4dadc46b1885fd

**Comment by jaley** [comment]
Hmm OK, so if it's deliberate it might just be worth flagging on release for other users. We can probably work around it until we redeploy with the next release.

**Comment by jafreck** [comment]
yeah, we decided to enforce a way to filter out non-aztk created batch pools from `aztk spark cluster list` as some users have other uses for their batch accounts. Unfortunately, that also means clusters made previous to that change will also be filtered out. We will definitely document this with the next release.

**Comment by jafreck** [comment]
@jaley @emlyn 

I think we are going to roll back this change. The change was meant to filter out job submission created clusters from `aztk spark cluster list` in part because the CLI UI was ugly. Instead, we will leave those job cluster's in, and just truncate their names.

After we roll this back, you will be able to see all of your `aztk` clusters as before.
